src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -23,6 +23,9 @@
 use JsonSerializable;
 use LogicException;
 
+/**
+ * @mixin Builder
```

Comment:
If you search for other `@mixin` usage throughout Laravel codebase, you will see every time a Full Qualified Class Name is used.

So I would change this to:

~~~php
/**
 * @mixin \Illuminate\Database\Eloquent\Builder
~~~